acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -1,6 +1,14 @@
 def initialize_repo_on_host(host, os, nightly)
   case os
   when :debian
+
+    # For openjdk8
+    if host['platform'].version == '8'
+      create_remote_file(host,
+                         "/etc/apt/sources.list.d/jessie-backports.list",
+                         "deb http://httpredir.debian.org/debian jessie-backports main")
```

Comment:
For puppetserver, we've been using `deb http://ftp.debian.org/debian jessie-backports main`.  That's apparently been working fine but would you suggest that we change to use what you've done here instead?